node/send_receive/lib/eventdata.js

```diff
@@ -6,9 +6,10 @@
 /**
  * @class EventData
  * @classdesc Constructs a {@linkcode EventData} object.
- * @param {String}  body   The event payload as a byte array.
+ * @param {object} link The receiver link for providing settlment methods.
+ * @param {object}  message The message object containing body and messageAnnotations.
  */
-function EventData(body, systemProperties) {
+function EventData(link, message) {
```

Comment:
in the IoT Hub SDK, settling messages is the responsibility of the client. I'd like the patterns to stay consistent across SDKs. Not sure which one we should favor at that point, I see pros and cons for both options, and these methods were moved around quite a bit (between client, receiver, message...). One thing is sure though, is that changing IoT Hub now is harder than Event Hubs... I'm going to give more thoughts to that.